pkg/rest/define/config.go

```diff
@@ -1,11 +1,36 @@
 package define
 
+import (
+	"github.com/crc-org/vfkit/pkg/config"
+)
+
+type VirtioNetResponse struct {
+	Nat            bool   `json:"nat"`
+	MacAddress     string `json:"macAddress"`
+	UnixSocketPath string `json:"unixSocketPath"`
+	Fd             int    `json:"fd"`
+}
+
+type DevicesResponse struct {
+	Input          []config.VirtioInput          `json:"input"`
+	GPU            []config.VirtioGPU            `json:"gpu"`
+	Vsock          []config.VirtioVsock          `json:"vsock"`
+	Blk            []config.VirtioBlk            `json:"blk"`
+	FS             []config.VirtioFs             `json:"fs"`
+	Rosetta        config.RosettaShare           `json:"rosetta"`
+	NVMe           []config.NVMExpressController `json:"nvme"`
+	Net            []VirtioNetResponse           `json:"net"`
+	Rng            bool                          `json:"rng"`
+	Serial         config.VirtioSerial           `json:"serial"`
+	USBMassStorage []config.USBMassStorage       `json:"usbMassStorage"`
```

Comment:
Can we reuse the same format as the preexisting json serialization code?
```
"devices":[{"devName":"virtio-blk","imagePath":"/virtioblk1","readOnly":false,"deviceIdentifier":""}]}
```
serialization/deserialization to this format should already be working, a new type should not be needed.
Or do you have strong reasons for having such a struct?

Reply:
https://github.com/crc-org/vfkit/pull/99 reuses the existing json format